source/speech/__init__.py

```diff
@@ -7,29 +7,52 @@
 """High-level functions to speak information.
 """ 
 
-from abc import ABCMeta, abstractmethod
-from six import with_metaclass
 import itertools
 import weakref
 import unicodedata
 import time
-import warnings
 import colors
-import globalVars
-from logHandler import log
 import api
 import controlTypes
-import config
 import tones
 import synthDriverHandler
 from synthDriverHandler import *
 import re
 import textInfos
-import queueHandler
 import speechDictHandler
 import characterProcessing
-import languageHandler
-from .commands import *
+from .commands import (
+	# Commands that are used in this file.
+	SpeechCommand,
+	PitchCommand,
+	LangChangeCommand,
+	BeepCommand,
+	EndUtteranceCommand,
+	CharacterModeCommand,
+)
+from .commands import (  # noqa: F401
+	# F401 imported but unused:
+	# The following are imported here because other files that speech.py
+	# previously relied on "import * from .commands"
+	# New commands added to commands.py should be directly imported only where needed.
+	SynthCommand,
+	IndexCommand,
+	SynthParamCommand,
+	BreakCommand,
+	BaseProsodyCommand,
+	VolumeCommand,
+	RateCommand,
+	PhonemeCommand,
+	BaseCallbackCommand,
+	CallbackCommand,
+	WaveFileCommand,
+	ConfigProfileTriggerCommand,
+)
+import types

```

Comment:
I realized that this is now a bit ambiguous, as we have types (a built-in python module) and speech.types. I assume you're referring to the built-in one here?

```suggestion
import types  # Python core module
```